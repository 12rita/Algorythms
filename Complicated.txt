log(tg(cos(x)))
1/(x^cos(x))
e^(log(ctg(x)))
arcctg(sin(e^x))