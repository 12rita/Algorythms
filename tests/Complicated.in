log(tg(cos(x)),sin(x))
1/(x^cos(x))
e^(log(ctg(x),x^x))
arcctg(sin(e^x))
