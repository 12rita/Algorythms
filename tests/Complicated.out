1/((1/(sin(x))*(cos(x))*ln(tg(cos(x)))-1/(tg(cos(x)))*(1/(cos(cos(x)))^2*(-sin(x)))*ln(sin(x)))/1/(tg(cos(x)))*(1/(cos(cos(x)))^2*(-sin(x)))^2)
-1/(-sin(x)*ln(x)+(1/(x))*cos(x))/(x^cos(x))^2
(e^(log(ctg(x)),x^x))*(1/(ln(log(ctg(x)),x)+(1/(log(ctg(x)),x)*())*x))
-1/(1+(sin(e^x))^2)*(cos(e^x)*((e^x)))
